tools/ldb_test.py

```diff
@@ -424,6 +424,15 @@ def testDumpLiveFiles(self):
         self.assertRunOK("put x3 y3", "OK")
         dumpFilePath = os.path.join(self.TMP_DIR, "dump2")
         self.assertTrue(self.dumpLiveFiles("--db=%s" % dbPath, dumpFilePath))
+        run_err_null("rm -rf %s" % dumpFilePath)
```

Comment:
Considering my other feedback, it might be better to test this functionality directly on OpenForReadOnly in one of our db unit tests. I don't think an ldb test is necessary if we have good unit tests on when OpenForReadOnly creates files/directories.

Reply:
Indeed, we should test it on OpenForReadOnly directly.